setup.py

```diff
@@ -1,5 +1,44 @@
 from setuptools import setup, find_packages
+from setuptools.command.develop import develop
 import os
+import sys
+
+try:
+    import jupyter_core.paths as jupyter_core_paths
+except:
+    jupyter_core_paths = None
+
+
+pjoin = os.path.join
+
+
+class DevelopCmd(develop):
+    prefix_targets = [
+        ("nbconvert/templates", 'gridstack'),
+        ("voila/templates", 'gridstack')
+    ]
+    def run(self):
+        target_dir = os.path.join(sys.prefix, 'share', 'jupyter')
+        if '--user' in sys.prefix:  # TODO: is there a better way to find out?
+            target_dir = jupyter_core_paths.user_dir()
+        target_dir = os.path.join(target_dir)
+
+        for prefix_target, name in self.prefix_targets:
+            source = os.path.join('share', 'jupyter', prefix_target, name)
+            target = os.path.join(target_dir, prefix_target, name)
+            target_subdir = os.path.dirname(target)
+            if not os.path.exists(target_subdir):
+                os.makedirs(target_subdir)
+            rel_source = os.path.relpath(os.path.abspath(source), os.path.abspath(target_subdir))
+            try:
+                os.remove(target)
```

Comment:
Shouldn't we let `os.symlink` fail instead of removing the target?

Reply:
it usually means there is an old directory or old symlink, effectively overwrite behavior.